layout/js/map.js

```diff
@@ -121,8 +123,18 @@ var get_state_value = (function() {
           }
         }(stateData);
     }
+    
+     if(stateHoverTimer){ 
+      clearTimeout(stateHoverTimer);
+     }
+     stateHoverTimer = setTimeout(function(){
```

Comment:
The issue with using the timer on this one is they will either have to leave their mouse perfectly still for 2 seconds or it will only report states on the edge of the svg (maybe even '' for states).  I think moving the timer to line 113ish would then trigger if they were over the same state for some period of time, and we might want to clear the timeout on mouseout completely.